src/stream/src/executor/hash_agg.rs

```diff
@@ -654,6 +734,8 @@ mod tests {
         pk_indices: PkIndices,
         extreme_cache_size: usize,
         executor_id: u64,
+        // TODO: should we use an enum here?
+        emit_immediate: bool,
```

Comment:
minor: What about directly using the generic `EmitPolicy` here?

Reply:
I'd prefer not, it's an internal structure, and is not ready to be used for other executors.